lib/Collaboration/Reference/TalkReferenceProvider.php

```diff
@@ -238,6 +240,7 @@ protected function fetchReference(Reference $reference): void {
 			'name' => $roomName,
 			'link' => $reference->getUrl(),
 			'call-type' => $this->getRoomType($room),
+			'message-id' => $messageId,
```

Comment:
instead of adding with null, we could also skip it in that case:

```php
		$referenceData = [
			'id' => $room->getToken(),
			'name' => $roomName,
			'link' => $reference->getUrl(),
			'call-type' => $this->getRoomType($room),
		];

		if ($messageId) {
			$referenceData['message-id'] = $messageId;
		}

		$reference->setRichObject('call', $referenceData);
```

Reply:
I like that, will change accordingly!